src/commonmark/en/content/developer/web-api.md

```diff
@@ -1262,41 +1256,46 @@ Multiple transformers can be used by repeating the transformer
 
 <!--DHIS2-SECTION-ID:webapi_field_transformers_examples-->
 
-Examples of transformer
-    usage.
+Examples of transformer usage.
 
-    /api/26/dataElements?fields=dataSets~size
+```
+/api/dataElements?fields=dataSets~size
 
-    /api/26/dataElements?fields=dataSets~isEmpty
+/api/dataElements?fields=dataSets~isEmpty
 
-    /api/26/dataElements?fields=dataSets~isNotEmpty
+/api/dataElements?fields=dataSets~isNotEmpty
 
-    /api/26/dataElements/ID?fields=id~rename(i),name~rename(n)
+/api/dataElements/ID?fields=id~rename(i),name~rename(n)
 
-    /api/26/dataElementGroups?fields=id,displayName,dataElements~paging(1;20)
-    
-    Include an array with IDs of the organisation units:
-    /api/31/categoryOptions.json?fields=id,organisationUnits~pluck
-    
-    Include an array with the names of the organisation units (collection only returns field name):
-    /api/31/categoryOptions.json?fields=id,organisationUnits~pluck[name]
+/api/dataElementGroups?fields=id,displayName,dataElements~paging(1;20)
+
+# Include array with IDs of organisation units:
+/api/33/categoryOptions.json?fields=id,organisationUnits~pluck
```

Comment:
Why use the version number here, when it is removed elsewhere? (same question for 3 lines below)

Reply:
Fixed.